src/components/Activity.tsx

```diff
@@ -1,4 +1,4 @@
-import { NiceP } from "./layout";
+import { NiceP } from "./layout/Misc";
```

Comment:
can this be changed to `"~components"`?

Reply:
I also find it much better and flexible for sure, it's exactly what I do in my projects

I didn't know how far I had to push it here

Reply:
?

Reply:
If you guys preferred a `./anything` or `~/components` whenever possible, which I personally prefer

I'm currently fixing the other imports, all good

Reply:
ahh ok got it :+1: I just saw that you had already made the imports consistent except for these few. Just pointed them out for the sake of consistency

Reply:
For sure, consistency is very important I agree !